client/packages/host/src/Site.tsx

```diff
@@ -247,13 +247,21 @@ export const Site: FC = () => {
                           <Navigate
                             replace
                             to={
-                              isGapsStore && isAndroid
-                                ? RouteBuilder.create(AppRoute.Coldchain)
+                              // isGapsStore is going to be refactored to support isGaps
+                              // This is a temporary fix until the refactor
+                              // isGapsStore is just a CSS breakpoint check atm
+                              // but is required on small devices
+                              (isGaps || isGapsStore) && isCentralServer
```

Comment:
Can you move this to a method please, with early returns, I alway struggle to read nested ternary

Reply:
Moved to its own method: https://github.com/msupply-foundation/open-msupply/pull/10067/changes/96c4248865b6a17e0c0e7e9949e4df8a0fcbd284